docs/thirdparty.rst

```diff
@@ -0,0 +1,77 @@
+Third Party Applications
+========================
+
+..
+    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+    If you're a developer and want to add your project/application to this list, please
+
+    1. adhere to the same structure
+    2. add your entry to the bottom of the list
+    3. at least provide the required fields (in the same order)
+       - Description (a brief text describing your application)
+       - Type (eg. Graphical User Interface, CLI wrapper, etc.)
+       - OS (please use the available substitutions)
+       - Author (if possible, include a link to the creator's Github/Gitlab profile, etc. or a contact email address)
+       - Website
+    4. use an image
+       - in the jpeg or png format
+       - with a static and reliable !!!https!!! URL (use Github or an image hoster like Imgur, etc.)
+       - with a reasonable size and aspect ratio
+       - with a decent compression quality
+       - that is not too large (at most 1 MiB allowed, the smaller the better)
+       - that is neutral and only shows your application
+    5. optionally add more fields like an URL to the source code repository or a larger description or features text
+
+    Please be aware that the Streamlink team may edit and remove your entry at any time.
+
+    Thank you! :)
+
+    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+
+.. raw:: html
+
+    <style>
+        .rst-content table.field-list td {
+            padding-top: 8px;
```

Comment:
I didn't want to change the theme stylesheets directly and had to add these inline styles, because the table content column somehow had a slightly negative vertical offset of 3px.

Caused by this CSS rule:
https://github.com/streamlink/streamlink/blob/0.7.0/docs/_themes/sphinx_rtd_theme_violet/static/css/theme.css#L4638
Which overrides this CSS rule:
https://github.com/streamlink/streamlink/blob/0.7.0/docs/_themes/sphinx_rtd_theme_violet/static/css/theme.css#L3301